### PR TITLE
Add additional typos

### PR DESCRIPTION
**Pull request by kianmeng**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -20549,6 +20553,7 @@ individaully->individually
 individauls->individuals
 individauly->individually
 individial->individual
+individiual->individual
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:20556]: Add `individiually->individually`?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -24119,6 +24124,7 @@ mirro->mirror
 mirroed->mirrored
 mirrorn->mirror
 mirrorred->mirrored
+mis->miss
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:24127]: Perhaps `mist` too, although not probable?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -25843,6 +25849,7 @@ nstalled->installed
 nstalling->installing
 nstalls->installs
 ntification->notification
+nto->not
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:25852]: `into`, `onto`, `unto`?

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -27737,6 +27745,7 @@ peprocessor->preprocessor
 per-interpeter->per-interpreter
 perade->parade
 peraphs->perhaps
+percenatge->percentages
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:27748]:
```
percenatge->percentage
percenatges->percentages
```

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -30381,6 +30390,7 @@ rapsberry->raspberry
 rarelly->rarely
 rarified->rarefied
 rasberry->raspberry
+rasbperry->raspberry
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:30393]:
```
rasbperries->raspberries
```

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -31605,6 +31615,7 @@ relaimed->reclaimed
 relaion->relation
 relaive->relative
 relaly->really
+relaoding->reloading
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:31618]:
```
relaod->reload
relaoded->reloaded
relaods->reloads
```

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -34359,6 +34370,7 @@ sertification->certification
 servce->service, serve,
 servced->serviced, served,
 servces->services, serves,
+servcie->service
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:34373]:
```
servcies->services
```

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -34680,6 +34692,7 @@ sigaled->signaled
 sigals->signals, sigils,
 siganture->signature
 sigantures->signatures
+sigature->signature
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:34695]:
```
sigatures->signatures
```

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -36940,6 +36953,7 @@ summmarised->summarised
 summmarization->summarization
 summmarized->summarized
 summmary->summary
+summurized->summarized
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:36956]:
```
summurize->summarize
summurizes->summarizes
summurizing->summarizing
```

File: codespell_lib/data/dictionary.txt [hunk 10]
```diff
@@ -37274,6 +37288,7 @@ swtiching->switching
 swtichover->switchover
 swtichs->switches
 sxl->xsl
+sxmbol->symbol
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:37291]:
```
sxmbolic->symbolic
sxmbols->symbols
```

File: codespell_lib/data/dictionary.txt [hunk 11]
```diff
@@ -9939,6 +9941,7 @@ correnspondents->correspondents
 corrensponding->corresponding
 corrensponds->corresponds
 corrent->correct, current,
+corrent->correct,current
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:9944]: Same as above, but missing space. Please remove.

File: codespell_lib/data/dictionary.txt [hunk 12]
```diff
@@ -456,6 +456,7 @@ accompagny->accompany
 accompagnying->accompanying
 accompained->accompanied
 accompanyed->accompanied
+accomponied->accompanied
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:459]:
```
accomponies->accompanies
accompony->accompany
accomponying->accompanying
```

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:24130]: Sorry, I actually meant
```
mis->miss, mist,
```

File: codespell_lib/data/dictionary.txt [hunk 13]
```diff
@@ -24219,6 +24228,7 @@ missunderstood->misunderstood
 missuse->misuse
 missused->misused
 missusing->misusing
+mist->miss
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:24231]: Please remove this one.

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:27753]: But why keep `percenatge->percentages`?

**Review by DimitriPapadopoulos — APPROVED** [review]

File: codespell_lib/data/dictionary.txt [hunk 14]
```diff
@@ -25843,6 +25852,7 @@ nstalled->installed
 nstalling->installing
 nstalls->installs
 ntification->notification
+nto->not, into, onto, unto
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:25855]: E   error nto: multiple corrections must end with trailing ","

**Comment by DimitriPapadopoulos** [comment]
Please wait for #2838 to be merged for error messages that make sense.

**Comment by DimitriPapadopoulos** [comment]
After rebasing, we get a decent error message:
```
codespell_lib/tests/test_dictionary.py:300: in test_dictionary_looping
    assert err not in other_err_dict, (
E   AssertionError: error 'nto' in dictionary dictionary_code.txt already exists in dictionary dictionary.txt
E   assert 'nto' not in {'1nd': ['1st'], '2rd': ['2nd'], '2st': ['2nd'], '3nd': ['3rd'], ...}
```
It appears `nto->not` already exists in `dictionary_code.txt`. Please remove from `dictionary.txt`.

**Comment by kianmeng** [comment]
:tada: :tada: :tada: :tada: :tada:

**Pull request merged**